Makefile

```diff
@@ -57,21 +63,24 @@ check_in_container: test_image
 		$(TEST_IMAGE) make check
 
 # deploy a pod with tests and run them
-check-inside-openshift: CONTAINER_ENGINE=docker
-check-inside-openshift: test_image
-	oc delete job packit-tests || :
+check-inside-openshift: CONTAINER_ENGINE ?= docker
+check-inside-openshift: worker_openshift_tests test_image
 	@# http://timmurphy.org/2015/09/27/how-to-get-a-makefile-directory-path/
 	@# sadly the hostPath volume doesn't work:
 	@#   Invalid value: "hostPath": hostPath volumes are not allowed to be used
 	@#   username system:admin is invalid for basic auth
 	@#-p PACKIT_SERVICE_SRC_LOCAL_PATH=$(dir $(realpath $(firstword $(MAKEFILE_LIST))))
-	oc process -f files/test-in-openshift.yaml | oc create -f -
-	oc wait job/packit-tests --for condition=complete --timeout=60s
-	oc logs job/packit-tests
-	# this garbage tells us if the tests passed or not
-	oc get job packit-tests -o jsonpath='{.status.conditions[?(@.type=="Complete")].status}' | grep True
+	$(AP) files/test-in-openshift-secrets.yaml
+	$(AP) files/check-inside-openshift.yaml
+
+check-inside-openshift-zuul: test_image
+	$(AP) files/check-inside-openshift.yaml
+
 
 # this target is expected to run within an openshift pod
 check-within-openshift:
 	/src-packit-service/files/setup_env_in_openshift.sh
 	pytest-3 -k test_update
+
+requre-purge-files:
+	requre-patch purge --replaces "requests.sessions%send:Date:str:Fri, 01 Nov 2019 13-36-03 GMT" --replaces 'requests.sessions%send:ETag:str:W/"1e51b8e1c48787a433405211e9e0fe61"' tests_requre/test_data/test_*/*.yaml
```

Comment:
ehm, this is pretty useless - the timestamp will be different all the time and I'm pretty sure that etag is valid only for a single github object; how about a regex?

Reply:
@TomasTomecek This is exactly the way how it works. It replaces all occurrences of ``Date`` and ``ETag``  in all ``'requests.sessions'->'send'`` by these pre-defined values, so that after regeneration files and calling this tool via make target should not change, unless there are other changes useful changes.
syntax is described in https://requre.readthedocs.io/en/latest/usages/postprocessing.html#command-line-tool

Reply:
ah, okay, got it